### PR TITLE
Add test case of pg binary value

### PR DESCRIPTION
**Pull request by xbkaishui**
Changes proposed in this pull request:
-  Add test case pg binary value

### CODE REVIEW

File: shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDateBinaryProtocolValueTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        assertThat(new PostgreSQLDateBinaryProtocolValue().read(payload), is(1L));
+    }
+    
+    @Test
+    public void assertWrite() {
+        Timestamp data = new Timestamp(System.currentTimeMillis());
+        new PostgreSQLDateBinaryProtocolValue().write(payload, data);
+        verify(payload).writeInt8(data.getTime());
+    }
+
```
Comment (tristaZero) [inline shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDateBinaryProtocolValueTest.java:56]: It is suggested to remove this redundant blank line.

File: shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDoubleBinaryProtocolValueTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        when(byteBuf.readDouble()).thenReturn(1D);
+        assertThat(new PostgreSQLDoubleBinaryProtocolValue().read(new PostgreSQLPacketPayload(byteBuf)), is(1D));
+    }
+    
+    @Test
+    public void assertWrite() {
+        new PostgreSQLDoubleBinaryProtocolValue().write(new PostgreSQLPacketPayload(byteBuf), 1D);
+        verify(byteBuf).writeDouble(1.0d);
+    }
+
```
Comment (tristaZero) [inline shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDoubleBinaryProtocolValueTest.java:54]: It is suggested to remove this redundant blank line.

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi @xbkaishui Except for some simple check style issues, this PR looks great.. :-)

File: shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/codec/PostgreSQLPacketCodecEngineTest.java [hunk 3]
```diff
@@ -65,6 +65,7 @@ public void assertDecode() {
         List<Object> out = new LinkedList<>();
         new PostgreSQLPacketCodecEngine().decode(context, byteBuf, out, 54);
         assertThat(out.size(), is(1));
+        assertThat(out.size(), is(1));
```
Comment (tuohai666) [inline shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/codec/PostgreSQLPacketCodecEngineTest.java:68]: Is it a duplicated line?

**Review by tuohai666 — CHANGES_REQUESTED** [review]

File: shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/codec/PostgreSQLPacketCodecEngineTest.java (hunk 3 above)
Comment (xbkaishui) [reply shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/codec/PostgreSQLPacketCodecEngineTest.java:68]: fixed

File: shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDateBinaryProtocolValueTest.java (hunk 1 above)
Comment (xbkaishui) [reply shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDateBinaryProtocolValueTest.java:56]: done

File: shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDoubleBinaryProtocolValueTest.java (hunk 2 above)
Comment (xbkaishui) [reply shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLDoubleBinaryProtocolValueTest.java]: done

**Review by tuohai666 — APPROVED** [review]

**Review by tristaZero — APPROVED** [review]

**Pull request merged**